api/src/main/java/org/pmiops/workbench/cohortbuilder/ParticipantCounter.java

```diff
@@ -33,6 +33,22 @@
                     "from `${projectId}.${dataSetId}.person` person\n" +
                     "where\n";
 
+    private static final String PARTICIPANT_STATUS_SQL_TEMPLATE =
+            "with race as\n" +
+                    "(select concept_id, concept_name FROM `${projectId}.${dataSetId}.concept` where domain_id = 'Race' or concept_id = 0),\n" +
```

Comment:
What's up with the concept_id = 0 stuff?
 
Is it necessary to define these tables? Seems like you could just select from concept below... we can assume that all concepts in the race_concept_id field are in the right domain.

Or if you just store IDs (see above), you don't have to bother with querying from concept at all.